src/Identicon/Identicon.php

```diff
@@ -101,4 +101,22 @@ public function getImageDataUri($string, $size = 64, $color = null, $backgroundC
     {
         return sprintf('data:%s;base64,%s', $this->generator->getMimeType(), base64_encode($this->getImageData($string, $size, $color, $backgroundColor)));
     }
+	
+	/**
+	 * Get the color of the Identicon
+     *
+     * Returns an array with RGB values of the Identicon's color. Colors may be NULL if no image has been generated
+     * so far (e.g., when calling the method on a new Identicon()).
+	 * 
+	 * @return array
+	 */
+	public function getColor() {
+		$colors = $this->generator->getColor();
```

Comment:
Yeah, but in this case the method should have been added to the interface before to prevent exceptions when using custom generators.. 

Anyways, i have added the method..